ui/app/src/components/DashboardBreadcrumbs.tsx

```diff
@@ -15,18 +15,41 @@ import { Breadcrumbs, Link, Typography } from '@mui/material';
 import { Link as RouterLink } from 'react-router-dom';
 
 interface DashboardBreadcrumbsProps {
-  dashboardName: string;
-  dashboardProject: string;
+  dashboardProject?: string;
+  dashboardName?: string;
 }
 
 function DashboardBreadcrumbs(props: DashboardBreadcrumbsProps) {
-  const { dashboardName, dashboardProject } = props;
+  const { dashboardProject, dashboardName } = props;
+
+  if (dashboardName) {
+    return (
+      <Breadcrumbs sx={{ fontSize: 'large' }}>
+        <Link underline={'hover'} variant={'h3'} component={RouterLink} to={'/'}>
+          Home
+        </Link>
+        <Link underline={'hover'} variant={'h3'} component={RouterLink} to={`/projects/${dashboardProject}`}>
+          {dashboardProject}
```

Comment:
the issue with that now is `dashboardProject` can be optional

Reply:
I added a check on `dashboardProject` + added a comment on the component